packages/content/src/messages/ArbitraryMessageContent.ts

```diff
@@ -0,0 +1,25 @@
+import { ISerializable, Serializable, serialize, type, validate } from "@js-soft/ts-serval";
+import { ContentJSON } from "../ContentJSON";
+
+export interface ArbitraryMessageContentJSON extends ContentJSON {
```

Comment:
yeah, this or push the actual content one level up by having something like `Map<string,...>` instead of `content`

Reply:
> We should be careful however, because the Arbitrary data could contain @ type which is not an "enmeshed @ type"

Why are we deleting @ type anyway?

Reply:
> We should be careful however, because the Arbitrary data could contain @ type which is not an "enmeshed @ type"

Yes! Please don't use JSONWrapper!